tasks/sync_repos.py

```diff
@@ -73,20 +73,20 @@ async def run_async(
                 blocking_timeout=5,
             ):
                 git = get_owner_provider_service(owner, using_integration)
+                synced_repoids = []
                 if using_integration:
                     with metrics.timer(f"{metrics_scope}.sync_repos_using_integration"):
-                        await self.sync_repos_using_integration(
+                        synced_repoids = await self.sync_repos_using_integration(
                             db_session, git, owner, username
                         )
                 else:
                     with metrics.timer(f"{metrics_scope}.sync_repos"):
-                        await self.sync_repos(
+                        synced_repoids = await self.sync_repos(
                             db_session, git, owner, username, using_integration
                         )
 
                 self.sync_repos_languages(
-                    ownerid=owner.ownerid,
-                    db_session=db_session,
+                    synced_repoids=synced_repoids or [],
```

Comment:
[nit] since you're setting synced_repoids above to `[]` i think the `or []` is not needed

Reply:
I added it here cause the sync repos fn has an explicit `return`, returning None, that I didn't want to change to `return []` so I rather added it here. I did get an error if I didn't supply a non-None value